build-config.lua

```diff
@@ -28,6 +28,9 @@ packtdszip  = true
 
 typesetcmds = typesetcmds or "\\AtBeginDocument{\\csname DisableImplementation\\endcsname}"
 
+typesetexe = "pdftex"
+typesetopts = "--fmt=pdflatex -interaction=nonstopmode"
+
```

Comment:
I'm guessing this is temporary?

Reply:
Not what I had in mind, but my knowledge of `l3build` is limited. What's
the problem here?

I added this (and the other block in the same file) so that when doing
`l3build doc` the format is build with the development version of
`l3kernel` rather than the TeXLive one.

This was necessary when building the documentation of `xfrac`, which
loads the dev version of `xtemplate` with the TL version of `l3kernel`,
and would break due to `\__kernel_quark_test_generate:NNNn` begin
undefined.

This change is _supposed_ to ensure that everything uses the dev version.

Reply:
I don't know enough about l3build and I don't understand this code; it just seems completely out of place in a pull request about quarks.  Perhaps @josephwright can help there?